app/models.py

```diff
@@ -3,12 +3,27 @@
 from werkzeug.security import generate_password_hash, check_password_hash
 
 
+# Create many-to-many mapping from groups to users using a helper table.
+groups = db.Table(
+    'groups',
+    db.Column("group_id", db.Integer, db.ForeignKey("group.id"), primary_key=True),
+    db.Column("user_id", db.Integer, db.ForeignKey("user.id"), primary_key=True),    
+)
+
+
 class User(UserMixin, db.Model):
     id = db.Column(db.Integer, primary_key=True)
     username = db.Column(db.String(64), index=True, unique=True)
     email = db.Column(db.String(120), index=True, unique=True)
     password_hash = db.Column(db.String(128))
     words = db.relationship("Slang", backref="author", lazy="dynamic")
+    groups = db.relationship(
+        "Group", 
+        secondary=groups, 
+        lazy="subquery",
+        backref=db.backref("groups", lazy=True),
+        overlaps="users, groups",
```

Comment:
I've been trying different permutations of this "overlaps=." parameter to suppress the CMD warning by flask upon navigation to index.html when logged in but to no avail